test/ng/httpSpec.js

```diff
@@ -35,7 +30,6 @@ describe('$http', function() {
       throw 'Unhandled exceptions trapped in $exceptionHandler!';
     }
 
-    $rootScope.$digest();
```

Comment:
Not necessary. `$httpBackend.verifyNoOutstandingExpectation()` will do a $digest first anyway (unless called with `false`, which is not the case here).